test_datasets/10x_pbmc_hto_GSE108313/download.R

```diff
@@ -1,13 +1,25 @@
 # Downloaded 10X data set, 'pbmc_hto_mtx.rds' and 'pbmc_umi_mtx.rds'
 # https://www.dropbox.com/sh/ntc33ium7cg1za1/AAD_8XIDmu4F7lJ-5sp-rGFYa?dl=0
+#
```

Comment:
Just wondering, was there anything wrong here or you just wanted to improve the script generally?

Reply:
The script is now fully automatic